File: https-quera.org-problemset-91712/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }

    public static void app() {
        Scanner scanner = new Scanner(System.in);

        int x1 = scanner.nextInt();
        int x2 = scanner.nextInt();
        int distance = Math.abs(x2 - x1);

        if (x2 > x1)
            for (int i = 0; i < distance; i++)
                System.out.print("R");
        else if (x1 > x2)
            for (int i = 0; i < distance; i++)
                System.out.print("L");
        else
            System.out.println("Saal Noo Mobarak!");
    }
}

File: https-quera.org-problemset-91712/Solution.py
def app():
    x1, x2 = map(int, input().split())
    distance = abs(x2 - x1)

    if x2 > x1:
        print("R" * distance)

    elif x1 > x2:
        print("L" * distance)

    else:
        print("Saal Noo Mobarak!")

if __name__ == "__main__":
    app()